bftengine/tests/messages/PrePrepareMsg_test.cpp

```diff
@@ -42,6 +46,47 @@ ClientRequestMsg create_client_request() {
                           concordUtils::SpanContext{spanContext});
 }
 
+std::string getRandomStingOfLength(size_t len) {
+  std::vector<char> alphabet{'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'p',
+                             'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd', 'e', 'f',
+                             'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
+                             'w', 'x', 'y', 'z', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', ' '};
+
+  std::mt19937_64 eng{std::random_device{}()};
+  std::uniform_int_distribution<> dist{0, static_cast<int>(alphabet.size())};
+  std::string res = "";
+  while (len > 0) {
+    res += alphabet[dist(eng)];
+    len--;
+  }
+  return res;
+}
+
+void create_random_client_requests(std::vector<std::shared_ptr<ClientRequestMsg>>& crmv, size_t numMsgs) {
+  crmv.clear();
+  std::mt19937_64 eng{std::random_device{}()};
+  std::uniform_int_distribution<> dist{12, 10000};
+  std::uniform_int_distribution<> msdist{0, 1000};
+
+  while (numMsgs > 0) {
+    uint64_t reqSeqNum = 100u + numMsgs;
```

Comment:
This `100u` is an implicit limit not exposed to the caller. If a caller passes numMsgs > 100, the size of the batch will be silently truncated. Why not instead remove the limit and count upwards?